proto/cel/expr/syntax.proto

```diff
@@ -199,10 +199,18 @@ message Expr {
   // return `result`
   // ```
   message Comprehension {
```

Comment:
In the message comment, could we include an example that involves `iter_var2`?

Reply:
Done.